diff -Naru a/build/cmake/lib/nanosvg.cmake b/build/cmake/lib/nanosvg.cmake
--- a/build/cmake/lib/nanosvg.cmake
+++ b/build/cmake/lib/nanosvg.cmake
@@ -16,9 +16,9 @@ elseif(wxUSE_NANOSVG)
     set(NANOSVG_INCLUDE_DIRS )
     set(wxUSE_NANOSVG_EXTERNAL_ENABLE_IMPL TRUE)
 
-    find_package(NanoSVG REQUIRED)
+    find_package(unofficial-nanosvg CONFIG REQUIRED)
 
-    foreach(TARGETNAME NanoSVG::nanosvg NanoSVG::nanosvgrast unofficial::nanosvg)
+    foreach(TARGETNAME unofficial::nanosvg::nanosvg)
         if(NOT TARGET ${TARGETNAME})
             continue()
         endif()
diff -Naru a/build/cmake/options.cmake b/build/cmake/options.cmake
--- a/build/cmake/options.cmake	2025-03-04 01:06:37.000000000 +0700
+++ b/build/cmake/options.cmake	2025-03-08 00:01:23.349521400 +0700
@@ -129,7 +129,7 @@
 wx_add_thirdparty_library(wxUSE_LIBJPEG JPEG "use libjpeg (JPEG file format)")
 wx_add_thirdparty_library(wxUSE_LIBPNG PNG "use libpng (PNG image format)")
 wx_add_thirdparty_library(wxUSE_LIBTIFF TIFF "use libtiff (TIFF file format)")
-wx_add_thirdparty_library(wxUSE_NANOSVG NanoSVG "use NanoSVG for rasterizing SVG" DEFAULT builtin)
+wx_add_thirdparty_library(wxUSE_NANOSVG unofficial-nanosvg "use NanoSVG for rasterizing SVG")
 
 wx_option(wxUSE_LIBLZMA "use LZMA compression" OFF)
 set(wxTHIRD_PARTY_LIBRARIES ${wxTHIRD_PARTY_LIBRARIES} wxUSE_LIBLZMA "use liblzma for LZMA compression")
diff -Naru a/build/cmake/wxWidgetsConfig.cmake.in b/build/cmake/wxWidgetsConfig.cmake.in
--- a/build/cmake/wxWidgetsConfig.cmake.in	2025-03-04 01:06:37.000000000 +0700
+++ b/build/cmake/wxWidgetsConfig.cmake.in	2025-03-08 00:02:15.807379100 +0700
@@ -1,5 +1,8 @@
 @PACKAGE_INIT@
 
+include(CMakeFindDependencyMacro)
+find_dependency(unofficial-nanosvg CONFIG)
+
 cmake_policy(PUSH)
 # Set policies to prevent warnings
 if(POLICY CMP0057)
