--- a/src/generic/bmpsvg.cpp
+++ b/src/generic/bmpsvg.cpp
@@ -25,12 +25,12 @@
 // initialize new submodules, if possible: if you get this error, it means that
 // your source tree doesn't contain 3rdparty/nanosvg and you should initialize
 // and update the corresponding submodule.
-#ifdef __has_include
+/*#ifdef __has_include
     #if ! __has_include("../../3rdparty/nanosvg/src/nanosvg.h")
         #error You need to run "git submodule update --init 3rdparty/nanosvg" from the wxWidgets directory.
         #undef wxHAS_SVG
     #endif
-#endif // __has_include
+#endif // __has_include*/
 
 #endif // wxHAS_SVG
 
