diff --git a/src/msw/evtloop.cpp b/src/msw/evtloop.cpp
index 34018e8bee..dc4859e6cb 100644
--- a/src/msw/evtloop.cpp
+++ b/src/msw/evtloop.cpp
@@ -84,6 +84,12 @@ bool wxGUIEventLoop::PreProcessMessage(WXMSG *msg)
             // only example of which I know of is the find/replace dialog) - then call
             // IsDialogMessage() to make TAB navigation in it work
 
+            // Except that we shouldn't call it for an undocumented message
+            // generated when using IME that it would consume, but that we must
+            // not actually process.
+            if ( msg->message == 0x0060 )
+                return false;
+
             // NOTE: IsDialogMessage() just eats all the messages (i.e. returns true for
             // them) if we call it for the control itself
             return hwnd && ::IsDialogMessage(hwnd, msg) != 0;
